Doxyfile

```diff
@@ -789,13 +789,13 @@ WARN_IF_DOC_ERROR      = YES
 # EXTRACT_ALL is set to YES then this flag will automatically be disabled.
 # The default value is: NO.
 
-WARN_NO_PARAMDOC       = NO
+WARN_NO_PARAMDOC       = YES
 
 # If the WARN_AS_ERROR tag is set to YES then doxygen will immediately stop when
 # a warning is encountered.
 # The default value is: NO.
 
-WARN_AS_ERROR          = NO
+WARN_AS_ERROR          = YES
```

Comment:
Enabling this has 2 potential changes. Either it'll cause CI to fail for lack of documentation somewhere (Which isn't necessarily bad, It's just that Most PRs are undocumented and I'm not sure if Crow's in code documentation is full enough to where this won't be more bad than good). Or it'll cause some builds to have no API reference (in the rare possibility that doxygen stops and returns 0).

Reply:
No, there was only 1 error and I fixed it.

Reply:
I understand. But the problem is that with the current configuration, if a PR causes a new warning and gets merged (since doxygen isn't built for PRs) CI will fail on master instead. I don't think it's a good idea to enable this.